test/integration/test-amphtml-ads.js

```diff
@@ -219,6 +220,7 @@ describe('AMPHTML ad on non-AMP page (inabox)', () => {
 
       it.configure()
         .skipSafari()
+        .skipFirefox()
```

Comment:
@alanorozco @lannka @powerivq I think this test should be skipped on all OSs until it's fixed. For example, this failure was on Edge: https://travis-ci.org/ampproject/amphtml/jobs/587583287#L2151

Reply:
Done in 5ed971024f10c38bcf805b0bd837cdfa544f9f9e. I'm going to merge this to unblock `master`.

Reply:
Yeah that's good.